src/flint/fmpz_mod.jl

```diff
@@ -56,13 +56,15 @@ is_unit(a::ZZModRingElem) = a.parent.n == 1 ? iszero(a.data) : isone(gcd(a.data,
 
 modulus(R::ZZModRing) = R.n
 
+characteristic(R::ZZModRing) = modulus(R)
+
+is_trivial(a::ZZModRing) = is_one(modulus(a))  # constructor ensures the modulus is > 0
```

Comment:
perhaps this makes it clearer?
```suggestion
is_trivial(a::ZZModRing) = is_one(modulus(a))  # this is correct because the constructor ensures the modulus is > 0
```